environments/common/inventory/groups

```diff
@@ -144,3 +144,10 @@ freeipa_client
 
 [lustre]
 # Hosts to run lustre client
+
+[dnf_repos:children]
+# Hosts to replace system repos with Pulp repos
+builder
+
+[pulp:children]
+# Hosts used to run Pulp API commands
```

Comment:
```suggestion
# Add builder to this group to enable automatically syncing of pulp during image build
```

Reply:
Will can you update the docs if necessary to be more in line with this comment pls?